### PR TITLE
Improve integration tests

### PR DESCRIPTION
**Pull request by madryk**
#38

### CODE REVIEW

File: iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java [hunk 1]
```diff
@@ -38,6 +50,8 @@ public IntegrationTestPropertiesReader() {
 	    
 	    integrationTestProperties.putAll(defaultProperties);
 	    integrationTestProperties.putAll(userProperties);
+	    
+	    createTempPropertiesFile();
```
Comment (lukdumi) [inline iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java]: This constructor doesn't look elegant. It implicitly reads properties, creates files. Apart from that - the class is not focused on one thing. Let's talk about it at our meeting.

File: iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java [hunk 2]
```diff
@@ -97,4 +127,27 @@ private Properties readUserProperties() {
 		return userProperties;
 	}
 	
+	private String resolveUserPropertiesFilePath() {
+		String userPropertiesPath = System.getProperty("connectionProperties");
```
Comment (lukdumi) [inline iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java]: I wonder if the "connectionProperties" name is not too generic. iisIntTestConnectionProperties?

File: iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java [hunk 3]
```diff
@@ -97,4 +127,27 @@ private Properties readUserProperties() {
...
+		if (userPropertiesPath == null) {
+			return DEFAULT_USER_PROPERTIES_PATH;
+		}
+		return userPropertiesPath;
+	}
+	
+	private void createTempPropertiesFile() {
```
Comment (lukdumi) [inline iis-core/src/test/java/eu/dnetlib/iis/core/IntegrationTestPropertiesReader.java]: I think it should be in a separate service method that takes Properties and file path as parameters.

File: iis-workflows/pom.xml [hunk 4]
```diff
@@ -686,47 +606,49 @@
...
 							</goals>
 							<configuration>
-								<skip>false</skip>
-								<forkMode>always</forkMode>
+								<redirectTestOutputToFile>true</redirectTestOutputToFile>
 								<argLine>-Xmx1024m</argLine>
 								<systemPropertiesVariables>
 									<hadoop.log.dir>/tmp</hadoop.log.dir>
 									<oozie.data.dir>${oozie.data.dir}</oozie.data.dir>
+									<connectionProperties>${connectionProperties}</connectionProperties>
```
Comment (lukdumi) [inline iis-workflows/pom.xml]: Information that if the connectionProperties is null then default connection properties (with location of it) will be used, would be useful.

**Comment by madryk** [comment]
I made all corrections.

**Comment by madryk** [comment]
I've applied all changes

**Pull request closed without merging**